server/routes/bookedtoday/unexpectedArrivals/index.ts

```diff
@@ -0,0 +1,67 @@
+import express, { RequestHandler, Router } from 'express'
+import SearchForExistingRecordController from './arrivals/searchForExistingRecordsController'
+import NoExistingRecordsFoundController from './arrivals/noExistingRecordsFoundController'
+import SingleExistingRecordFoundController from './arrivals/singleExistingRecordFoundController'
+import MultipleExistingRecordsFoundController from './arrivals/multipleExistingRecordsFoundController'
+import authorisationForUrlMiddleware from '../../../middleware/authorisationForUrlMiddleware'
+import asyncMiddleware from '../../../middleware/asyncMiddleware'
+import { Services } from '../../../services'
+import Role from '../../../authentication/role'
+import validationMiddleware from '../../../middleware/validationMiddleware'
+import searchForExistingRecordsValidator from './arrivals/validation/searchForExistingRecordsValidation'
+
+export default function routes(services: Services): Router {
+  const router = express.Router()
+
+  const get = (path: string, handlers: RequestHandler[], authorisedRoles?: Role[]) =>
+    router.get(
+      path,
+      authorisationForUrlMiddleware(authorisedRoles),
+      handlers.map(handler => asyncMiddleware(handler))
+    )
+
+  const post = (path: string, handlers: RequestHandler[], authorisedRoles?: Role[]) =>
+    router.post(
+      path,
```

Comment:
maybe include the subfolders in the path parameter to reduce repetition below similar as we do in searchforexisting/index.ts :
/manually-confirm-arrival/search-for-existing-record${path},